src/styles/mixins.less

```diff
@@ -0,0 +1,17 @@
+// TODO: word-break mixins can be removed once the web console mixin is migrated to commons so these are accessible.
```

Comment:
These need to be moved from origin-web-console into origin-web-common instead of here. I'd rather do that in a follow on than have them defined twice.